clients/spark/build.sbt

```diff
@@ -1,6 +1,6 @@
-lazy val projectVersion = "0.15.0"
+lazy val projectVersion = "0.15.2"
```

Comment:
@nopcoder Changing the default Hadoop version doesn't require a minor version bump?